ios/RNDatePickerManager.h

```diff
@@ -8,7 +8,14 @@
 #import <React/RCTConvert.h>
 #import <React/RCTViewManager.h>
 
+#ifdef RCT_NEW_ARCH_ENABLED
+#import "RNDatePickerSpecs.h"
```

Comment:
This worked better for me, not sure if there is a difference.
```suggestion
#import <RNDatePickerSpecs/RNDatePickerSpecs.h>
```